javascript-and-json/main.js

```diff
@@ -0,0 +1,27 @@
+const library = [
+  {
+    isbn: '978-0060935467',
+    title: 'To Kill a Mockingbird',
+    author: 'Harper Lee',
+  },
+  {
+    isbn: '978-0060256678',
+    title: 'Where the Sidewalk Ends',
+    author: 'Shel Silverstein',
+  },
+  {
+    isbn: '978-0141439563',
+    title: 'Great Expectations',
+    author: 'Charles Dickens',
+  },
+];
+
+console.log('typeof library:', typeof library);
+
+console.log('JSON.stringify typeof library:', typeof JSON.stringify(library));
+
+const student = JSON.stringify({ id: '98042423', name: 'Paxton' });
```

Comment:
Create a String in JSON format holding data representing **a student** with a Number id and a String name.

You will create a string manually in JSON format:

eg. `const studentJSON = '{"id":1,"name":"Jeff Jefferson"}';`

Reply:
thank you for the feedback - I corrected this mistake here and pushed the updates here.